src/Compiler/TypedTree/TypedTreePickle.fs

```diff
@@ -1435,6 +1436,8 @@ let p_trait_sln sln st =
          p_byte 6 st; p_tup5 p_ty (p_option p_ILTypeRef) p_ILMethodRef p_tys p_ty (a, b, c, d, e) st
     | FSMethSln(a, b, c, Some d) ->
          p_byte 7 st; p_tup4 p_ty (p_vref "trait") p_tys p_ty (a, b, c, d) st
+    | ILFieldSln(a, b, c, d) ->
```

Comment:
You;ll also need a producing case in `u_trait_sln`

Reply:
Yep, couple of things are missing here